tests/test_classification.py

```diff
@@ -199,8 +204,8 @@ def check_label_index_order_with_pos_label_str(use_api=True):
         # if we are using the API, create a learner instance directly
         # otherwise run a configuration file to train a learner
         if use_api:
-            clf = Learner('LogisticRegression', pos_label_str=pos_label_str)
-            clf.train(train_fs, grid_search=False)
+        clf = Learner('LogisticRegression', pos_label_str=pos_label_str)
```

Comment:
should not this be indented?

Reply:
Weird, not sure how that got unindented.